### PR TITLE
Brett

### PR DESCRIPTION
**Pull request by PolarisStarnor**
US 04.01.01 profile with unique username and contact information#14
US 01.02.01 see added QR codes#2

Created a player class to represent users of QReatures.
Methods for comparing usernames, device ids, and contact information to those present in the database (but haven't been tested.)
Profiles now draw from firebase to access creatures. Player-specific creatures won't be implemented until we figure out how Players and Creatures link together in Firebase.

### CODE REVIEW

**Review by sepniojr — APPROVED** [review]

File: code/app/src/main/java/com/example/qrcodesfornoobs/Player.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+import android.provider.ContactsContract;
+
+import java.util.ArrayList;
+
+public class Player {
+
+    private String username;
+    private String device;
+    private ContactsContract.CommonDataKinds.Email contact;
```
Comment (thomassuu) [inline code/app/src/main/java/com/example/qrcodesfornoobs/Player.java:11]: probably remove this attribute cuz a post on eclass said only username is mandatory so we'll keep it simple for now

File: code/app/src/main/java/com/example/qrcodesfornoobs/Player.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        this.device = device;
+    }
+
+    public Player(String username, String device, ContactsContract.CommonDataKinds.Email contact) {
+        this.username = username;
+        this.device = device;
+        this.contact = contact;
+    }
+
+    public void addCreature(Creature creature) {
```
Comment (thomassuu) [inline code/app/src/main/java/com/example/qrcodesfornoobs/Player.java:25]: add a TODO to call db

File: code/app/src/main/java/com/example/qrcodesfornoobs/Player.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+    public String getUsername() {
+        return username;
+    }
+
+    public String getDevice(){
+        return device;
+    }
+
+    public ContactsContract.CommonDataKinds.Email getContact() {
```
Comment (thomassuu) [inline code/app/src/main/java/com/example/qrcodesfornoobs/Player.java:45]: remove this as email is no longer needed

**Review by thomassuu — APPROVED** [review]
change login function when db format is finalized

**Pull request merged**